dev/com.ibm.ws.componenttest/src/componenttest/app/JavaInfo.java

```diff
@@ -0,0 +1,173 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package componenttest.app;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A class used for identifying properties of a JDK
+ */
+public class JavaInfo {
```

Comment:
I suggest changing to be more like com.ibm.ws.kernel.service.util.JavaInfo with just instance instead of a Map of JavaInfo's.  Also remove the vendor stuff since that is all deprecated.  Let's not add it into this new one.